scripts/scripts.js

```diff
@@ -209,6 +209,26 @@ function decorateExternalImages(ele, deliveryMarker) {
   });
 }
 
+/**
+ * Decorates all images in a container element and replace media urls with delivery urls.
+ * @param {Element} main The container element
+ */
+function decorateImages(main) {
+  const pictureElements = main.querySelectorAll('picture');
+  [...pictureElements].forEach((pictureElement) => {
+    const imgElement = pictureElement.querySelector('img');
+    const alt = imgElement.getAttribute('alt');
+    try {
+      const deliveryObject = JSON.parse(decodeURIComponent(alt));
+      const { deliveryUrl, altText } = deliveryObject;
+      const newPictureElement = createOptimizedPicture(deliveryUrl, altText);
```

Comment:
Can we use `createOptimizedPicture` as it is? It will make the URL relative by removing the Delivery domain. 

Please share the test page to confirm it is working.

Reply:
looks good, ignore my previous comment. Just we can have the following imho:
1. Rename this method to something like `decorateDeliveryImages` or something on those lines
2. have a `if` check to only process if the Image was pasted from Delivery (i.e. alt text represents a JSON object and it has deliveryUrl)